Content.Server/Kitchen/EntitySystems/MicrowaveSystem.cs

```diff
@@ -517,10 +559,22 @@ public override void Update(float frameTime)
             base.Update(frameTime);
 
             var query = EntityQueryEnumerator<ActiveMicrowaveComponent, MicrowaveComponent>();
+
```

Comment:
Here the declaration of `query` above is closely associated with this while loop, so you can get rid of this extra whitespace.